cpp/src/sender.cpp

```diff
@@ -26,10 +26,12 @@
 #include "proton/source.hpp"
 #include "proton/target.hpp"
 #include "proton/tracker.hpp"
+#include "types_internal.hpp"
```

Comment:
This include is in the wrong block of includes - it is purely internal so should in the bottom block